Configuration/Skimming/python/PDWG_BPHSkim_cff.py

```diff
@@ -0,0 +1,72 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Reconstruction_cff import *
+
+# muons with trigger info
+from MuonAnalysis.MuonAssociators.patMuonsWithTrigger_cff import *
+selectedMuons = cms.EDFilter('PATMuonSelector',
```

Comment:
@kfjack - here and in the other modules in this skim I would ask that you change the name to be something less generic to avoid potential name clashes. This would affect selectedMuons, selectedTracks and PhotonCandidates.